src/components/ImageView/index.js

```diff
@@ -197,14 +203,15 @@ class ImageView extends PureComponent {
             return (
                 <View
                     style={[styles.imageViewContainer, styles.overflowHidden]}
+                    onLayout={this.onContainerLayoutChanged}
                 >
                     <Image
                         source={{uri: this.props.url}}
-                        style={[
+                        style={this.state.zoomScale === 0 ? undefined : [
                             styles.w100,
                             styles.h100,
-                        ]}
-                        resizeMode="contain"
+                        ]} // Hide image until zoomScale calculated
+                        resizeMode={this.state.zoomScale > 1 ? 'center' : 'contain'} // For big dimension images 'contain' works much effective
```

Comment:
Improve this comment. You should tell how that is more effective and what is the effect.

Reply:
Best thing to do would be find why image is all stretched on mWeb while it work great on Web.

Reply:
That code doesn't work on the web. 
The Image component is inside the Pressable component on desktop and Web.

Reply:
I meant to ask why image is all stretched on mWeb while it work great on Web for `contain` value. 
Why `contain` is not working on mWeb same as web.

Reply:
On the web side, there is a `pressable` component on the upper level of the `image` component. When small size photos come, it changes the style information and keeps it in the middle of the screen. It does this by not giving the entire container space to the Image component.
But in mWeb, Image component has all the space. I can do it like on the web, if I write the code below, it works. But I think we would have done an extra calculation. The current version of PR is actually doing this job for us.

```js
<Pressable
        style={{
                            ...StyleUtils.getZoomSizingStyle(this.state.isZoomed, this.state.imgWidth, this.state.imgHeight, this.state.zoomScale,
                                this.state.containerHeight, this.state.containerWidth),
                            ...StyleUtils.getZoomCursorStyle(this.state.isZoomed, this.state.isDragging),
                            ...this.state.isZoomed && this.state.zoomScale >= 1 ? styles.pRelative : styles.pAbsolute,
                            ...styles.flex1,
                        }}
                    >
                        <Image
                            source={{uri: this.props.url}}
                            style={this.state.zoomScale === 0 ? undefined : [
                                styles.w100,
                                styles.h100,
                            ]} // Hide image until zoomScale calculated
                            resizeMode="contain"
                            onLoadStart={this.imageLoadingStart}
                            onLoadEnd={this.imageLoadingEnd}
                        />
                        {this.state.isLoading && (
                            <FullscreenLoadingIndicator
                                style={[styles.opacity1, styles.bgTransparent]}
                            />
                        )}
                    </Pressable>
```

Reply:
Ok, then leave this